tests/ops/api/v1/endpoints/test_manual_webhooks.py

```diff
@@ -104,9 +112,21 @@ def url(self, integration_manual_webhook_config) -> str:
     def payload(self):
         return {
             "fields": [
-                {"pii_field": "First name", "dsr_package_label": None},
-                {"pii_field": "Last name", "dsr_package_label": "last_name"},
-                {"pii_field": "Order number", "dsr_package_label": "order_number"},
+                {
+                    "pii_field": "First name",
+                    "dsr_package_label": None,
+                    "data_categories": ["user.name"],
+                },
+                {
+                    "pii_field": "Last name",
+                    "dsr_package_label": "last_name",
+                    "data_categories": ["user.name"],
```

Comment:
Can we add one test that checks for handling multiple data categories?  Something like 3-4 of them